modelservice/src/main/java/com/expedia/adaptivealerting/modelservice/service/ModelService.java

```diff
@@ -28,9 +28,9 @@
 
     List<ModelDto> getModels(String metricKey);
 
-    void addModelParams(ModelParams modelParams);
+    void addHyperparams(String modelUUID, Hyperparams hyperparams);
```

Comment:
Can you explain why we are "adding" hyperparams here? Why wouldn't we just save the whole model with its hyperparams as a single action? And anyway, even if we were to do this as a two actions for whatever reason, we would be _setting_ the hyperparameters, not _adding_ them. Adding connotes augmenting a collection. Here we are setting the hyperparams on the model. But please prefer just saving the model with its hyperparams in one action unless there is some reason to do it as two actions.